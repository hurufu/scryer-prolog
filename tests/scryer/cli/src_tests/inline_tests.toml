args = [
    "-f",
    "--no-add-history",
    "src/tests/inline_tests.pl",
    "-g", "run",
    "-g", "halt"
]
