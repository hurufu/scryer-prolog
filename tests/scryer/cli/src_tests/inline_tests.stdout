test(A) :-
   A is 2.
